translations/ui-receiving/ca.json

```diff
@@ -129,5 +129,6 @@
     "piece.actions.selectAll": "Select all pieces",
     "piece.actions.select": "Select piece",
     "piece.confirmReceiving.title": "Order closed",
-    "piece.confirmReceiving.message": "The order linked to this Title is closed. Are you sure you want to receive this piece(s)?"
+    "piece.confirmReceiving.message": "The order linked to this Title is closed. Are you sure you want to receive this piece(s)?",
+    "titles.orderWorkflow": "Order status"
```

Comment:
@dltj here we have the right value, "Order status"

Reply:
I manually updated these entries in Lokalise.  There is a merge-to-GitHub process happening now.